packages/nouns-contracts/test/foundry/NounsDAOLogic/NounsDAOAdmin.t.sol

```diff
@@ -22,6 +22,121 @@ contract NounsDAOLogicAdminTest is NounsDAOLogicBaseTest {
 
     address[] tokens;
 
+    function test__setVotingDelay_onlyAdmin() public {
```

Comment:
missing test for:
- `_setForkParams`
- `_withdraw`
- `_burnVetoPower`
- `_acceptVetoer`
- `_setPendingVetoer`
- `_setPendingAdmin` - check only admin can call
- `_acceptAdmin` - check only pending admin can call

Reply:
* `_setForkParams`: I decided to skip because we have tests for all the individual params, but I gather that you think it's important so I don't mind adding anyway.
* `_withdraw`: tested in the file Withdraw.t.sol
* `_burnVetoPower`: tested in the file Veto.t.sol
* `_acceptVetoer`: tested in the file Veto.t.sol
* `_setPendingVetoer`: tested in the file Veto.t.sol
* `_setPendingAdmin`: will add onlyAdmin test
* `_acceptAdmin`: will add the only pending admin check